server.js

```diff
@@ -27,6 +30,53 @@ app.get('/', function (req, res) {
      res.sendFile(__dirname + '/src/index.html');
 });
 
+//Marketing endpoint
+app.get('/marketing', function(req,res){
+    const con = mysql.createConnection(databaseConnectionOptions);
+    const date_time_format = "YYYY-MM-DD hh:mm:ss";
+    var today= moment.utc();
+    //Starting date of this week
+    var thisWeek = today.startOf("week");
```

Comment:
Tested with real data and found an issue. clone `today `as well for `thisWeek `cause  the same object is being passed around and manipulated. `thisWeek `becomes the same value as `lastMonth`